app/app.js

```diff
@@ -150,4 +152,13 @@ app.get('/robots.txt', function (req, res) {
   res.send('User-agent: *\nDisallow: /')
 })
 
+// // expose the error in combination with nunjucks env options
```

Comment:
Nit: // //

Also, can you explain in the comment what this is doing, the setTimeout I'm not sure why it's required